js/client.js

```diff
@@ -2741,15 +2741,27 @@ function toId() {
 		initialize: function (data) {
 			var buf = '';
 			buf = '<p>Your replay has been uploaded! It\'s available at:</p>';
-			buf += '<p><a href="https://' + Config.routes.replays + '/' + data.id + '" target="_blank" class="no-panel-intercept">https://' + Config.routes.replays + '/' + data.id + '</a></p>';
-			buf += '<p><button class="autofocus" name="close">Close</button></p>';
+			buf += '<p> <a id="replayLink" href="https://' + Config.routes.replays + '/' + data.id + '" target="_blank" class="no-panel-intercept">https://' + Config.routes.replays + '/' + data.id + '</a></p>';
+			buf += '<p><button class="autofocus" name="close">Close</button> <button name="copyReplayLink">Copy</button><p>';
```

Comment:
```suggestion
			buf += '<p> <a id="replayLink" href="https://' + Config.routes.replays + '/' + data.id + '" target="_blank" class="no-panel-intercept">https://' + Config.routes.replays + '/' + data.id + '</a> <button name="copyReplayLink">Copy</button></p>';
			buf += '<p><button class="autofocus" name="close">Close</button><p>';
```

I think putting Copy and Close on the same row will screw with muscle memory. This should be safer.